plugin/batchedNMSPlugin/batchedNMSPlugin.cpp

```diff
@@ -68,7 +68,7 @@ Dims BatchedNMSPlugin::getOutputDimensions(int index, const Dims* inputs, int nb
     ASSERT(nbInputDims == 2);
     ASSERT(index >= 0 && index < this->getNbOutputs());
     ASSERT(inputs[0].nbDims == 3);
-    ASSERT(inputs[1].nbDims == 2);
+    ASSERT(inputs[1].nbDims == 2 || (inputs[1].nbDims == 3 && inputs[1].d[3] == 1));
```

Comment:
Isn't it (inputs[1].nbDims == 3 && inputs[1].d[2] == 1)?

Reply:
Yes, thanks for catching that.